src/test/java/net/woody/pieces/PawnTest.java

```diff
@@ -12,15 +12,17 @@
     @DisplayName("흰색 폰과 검은색 폰이 생성되어야 한다")
     void createWhiteAndBlackPawns() {
         assertAll(
-                () -> verifyPawn(Pawn.WHITE_COLOR),
-                () -> verifyPawn(Pawn.BLACK_COLOR));
+                () -> verifyPawn(Color.WHITE.colorName()),
+                () -> verifyPawn(Color.BLACK.colorName())
+        );
     }
 
     @Test
     @DisplayName("기본 생성자로 폰을 생성할 때, 흰색 폰이 생성되어야 한다")
     void createWithDefaultPawn() {
         Pawn pawn = new Pawn();
-        assertThat(Pawn.WHITE_COLOR).isEqualTo(pawn.getColor());
+        assertThat(Color.WHITE).isEqualTo(pawn.getColor());
```

Comment:
```suggestion
        assertThat(pawn.getColor())
            .isEqualTo(Color.WHITE);
```

호눅스님이 수업때 말씀하셨던 것 것처럼, actual 을 앞에두고, expected 를 뒤로 써주시는 게 좋습니다.
intelliJ 의 자동완성에서 보여주는 파라미터 명을 가능한 따라주시면 좋아요

Reply:
완전 자세하게 봐주셔서 감사드려요! 
이 부분은 제가 제대로 확인하지 않았네요!! 수정하도록 하겠습니다 감사합니다😀😊